showcase/src/app/home/_home.component.scss

```diff
@@ -1,16 +1,8 @@
-//custom title for the header. Keep in mind that we define the font-size and color here because it is an exceptional case.
-//Should this style be reused elsewhere, it would be better to create a new style in stark-ui
-.about-title {
-  font-size: 30px;
-  margin-left: 15px;
-  color: mat-color($grey-palette, 100);
-}
-
 //set the header image
 .about-background {
-  background: url("/assets/images/banner/angular-banner.png") no-repeat center center;
-  background-size: cover;
-  height: 170px;
+  background: mat-color($primary-palette, 900) url("/assets/stark-core/logo/stark_logo_transparent.svg") no-repeat center center;
```

Comment:
Is this working? I don't think that this path is configured anywhere.... the right path should be the one you define in the `angular.json` file of the stark-core ;)

Reply:
I can confirm, it is working too 😊